data/json/items/gun/223.json

```diff
@@ -45,6 +45,31 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "db15",
+    "copy-from": "rifle_semi",
+    "looks_like": "skorpion_61",
+    "type": "GUN",
+    "name": { "str": "DB-15" },
```

Comment:
Call it a Pistol AR? Diamondback didn't pioneer this, it just describes a type of AR. Singling it out as the DB-15 is very odd.

Reply:
I'd argue that pistol AR is the generic name, and every single gun (outside of generic guns) uses brand names, so using the brand name seems correct.

Reply:
So we're supposed to add Aero precision brand Ar-15, Radical firearms AR pistols, PSA AR pistols, BCM AR pistols, Daniel Defense AR pistols, Anderson AR Pistols, Doublestar AR pistols, Springfield Armory SAINT,  CBC industries AR pistols...

Reply:
What is the difference between and AR pistol made by Diamondback and a rifle made by Remington? This is just one specific model by one random company. This doesn't have to imply that this is the only or even the best AR pistol on the market.

Reply:
If someone wants to, i suppose you could add those weapons to the game if you wanted. We tend to draw the line at a certain point because people can't be bothered to add more guns of the same type, but switching to generic names here and now seems like a weird place.

Reply:
There isn't a difference. If you add specifically the diamondback model, then you're looking at a very small marketshare, and this gun would go back to being very rare. Make it general, and they're common.

Reply:
That's clear. My research was to simply google the most common 7.5 inch AR pistols on the market, and this was the first one roughly fitting the pictures used in volume calculations you gave earlier.

I also don't think that generalization is a good idea here. I can change the name to match the most common manufacturer, then lower the spawn weights to match the actual number of guns on the market, if you can provide some guesses about that.

Reply:
I replaced the DB-15 with a generic AR pistol item.